cli/.eslintrc

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
I wonder if we can turn this off at the .eslintrc.yml file in root.
It is set to error currently but there are a bunch of console.log's throughout the repo. Doesn't feel like it's adding much value period.

Reply:
Done.  Going to need another approval here.